docs/output.md

```diff
@@ -28,14 +28,14 @@ Stand-alone graphical outputs are placed in this directory. They may be useful i
   - `qc/`: Directory containing quality control plots from initial processing e.g. DESeq2
     - `*.png`
   - `exploratory/`: Directory containing standalone plots from exploratory analysis. Plots are stored in directories named for the main coloring variable used.
-    - `[coloring variable]/png/boxplot.png`: Boxplot visualisation of abundance distributions
-    - `[coloring variable]/png/density.png`: Density visualisation of abundance distributions
+    - `[coloring variable]/png/boxplot.png`: Boxplot visualization of abundance distributions
+    - `[coloring variable]/png/density.png`: Density visualization of abundance distributions
     - `[coloring variable]/png/pca2d.png`: 2-dimensional PCA plot
     - `[coloring variable]/png/pca3d.png`: 3-dimensional PCA plot
     - `[coloring variable]/png/sample_dendrogram.png`: A sample clustering dendrogram
     - `[coloring variable]/png/mad_correlation.png`: Outlier prediction plots using median absolute deviation (MAD)
   - `differential/`: Directory containing standalone plots from differential analysis. Plots are stored in directories named for the associated contrast.
-    - `[contrast]/png/volcano.png`: Volcano plots of -log(10) p value agains log(2) fold changes
+    - `[contrast]/png/volcano.png`: Volcano plots of -log(10) p value against log(2) fold changes. Each dot in the plot represents one gene. The higher genes in the volcano plot are highly significant, and the genes located in right or left side of the plot are highly dysregulated.
```

Comment:
Sorry, the output.md is not a great place for extensive explanation of plots. Also, 'volcano plot' has special meaning most people understand.

Reply:
sorry, I disagree here. https://nf-co.re/eager/2.5.1/docs/output is for me a prime example of pipeline documentation including thorough explanations of the plots. (even though I agree that vulcano plots are quite established, a bit of explanation doesn't hurt imo)